IMC_WebDev/FinalBlog.md

```diff
@@ -0,0 +1,224 @@
+```
+You may notice that there is a &lt;/title&gt; tag after our title’s name. This is called a closing tag. Closing tags have a “/” before they state which tag they are closing. This just tells our editor that we are no longer editing this tag.
+
+To create our body section, simply type body on your next line.
+
+Now, let’s make our classes! Let’s think about what we want our city to look like.
+We know that we want the following objects in our city:
+The ground
+A skyscraper
+A smaller building
+A road
+A bridge
+An ocean
+A waterfall
+A car
+
+We have to make classes for these objects so that we can create them later on. Classes are simply declarations that we use to establish the fact that we are going to create an object. Then, we can actually create it in CSS!
+
+We create classes using the &lt;div&gt; tag. Like this:
+```html
+<body>
+    <div class = "ground">
+```
+As you can see, we just created a class called “ground”. Now our editor knows that we are planning to create an object.
+
+We do the same thing with all of our other objects. When you are done, it should look something like this: 
+```html
+<body>
+    <div class = "ground">
+      <div class = "bottomSide"></div> 
+        <div class = "bottomFront"></div> 
+        <div class = "building1">
+          <div class = "wall1"></div>
+          <div class = "wall2"></div>
+          <div class = "roof"></div>
+          <div class = "railing1"></div>
+          <div class = "railing2"></div>
+        </div>
+        <div class = "building2">
+          <div class = "wall3"></div>
+          <div class = "wall4"></div>
+          <div class = "roof2"></div>
+        </div>
+        <div class = "road1"></div>
+        <div class = "road2"></div>
+      </div>
+    
+    	<div class = "ocean"></div>
+    	<div class = "ramp1"></div>
+      <div class = "ramp2"></div>
+      <div class = "attempt1"></div>
+    
+      <div class = "bridge">
+      	<div class = "railing1"></div>
+      </div>
+      
+      <div class = "railing2"></div>
+      
+    
+    	<div class = "waterfall"></div>
+    
+    	<div class = "ground2">
+    		<div class = "grassArea"> <!--GRASS AREA-->
+    			<div class = "treeTrunks"></div> 
+          <div class = "triangleTops"></div>
+          <div class = "circleTops"></div> 
+    		</div>
+        <div class = "bottomFront"></div> 
+    	</div>
+      
+      <div class = "car">
+        <div></div>
+        <div></div>
+      </div>
+    
+    	<div class = "boat">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+    
+    	<div class = "sun"></div>
+  </body>
+```
+
+This looks like a huge chunk of code right now, but let’s break it down!
+
+HTML works as the canvas for all of our objects, so for every object in our city, we create a div class for it. Once we do that, we can edit those divs using our CSS, which is like our paint! It allows us to control how the objects appear on the site, like color, layout, etc.
```

Comment:
I added a sentence here to further explain the analogy between HTML/CSS and canvas/paint; feel free to change or edit to make it more technically accurate!